modules/local/krona.nf

```diff
@@ -8,15 +8,16 @@ process KRONA {
 
     input:
     tuple val(meta), path(report)
-    path  "taxonomy/taxonomy.tab"
+    path(taxonomy_file)
 
     output:
-    path "*.html"       , emit: html
-    path "versions.yml" , emit: versions
+    tuple val(meta), path("*.html") , emit: html
+    path "versions.yml"             , emit: versions
 
     script:
+    taxonomy_folder = taxonomy_file.toRealPath().getParent()
     """
-    ktImportTaxonomy "$report" -tax taxonomy
+    ktImportTaxonomy "$report" -tax ${taxonomy_folder}
```

Comment:
'm a bit concerned by 'toRealPath()': what does this do exactly - does it try to replace a symlink with a 'real' path? This _may_ be problematic  cloud situations (I'm not an expert but has been brought up with me in different cases along similar lines). 

Why not leave it as passing the file directly and running just `.getParent()`? or does that not work with symlinks?

Could you just not supply the directory to the parameter rather than the `.tab` file itself?

Reply:
ndeed, I agree. I will eliminate the usage of `toRealPath()`.

The rationale behind allowing users to indicate a file rather than directly selecting a folder is that the output from the Krona DB download process generates a file. Hence, I attempted to maintain consistency in this approach.

Reply:
OK that's reasonable!